spec/factories/ahoy.rb

```diff
@@ -0,0 +1,18 @@
+FactoryBot.define do
+  factory :ahoy_message, class: "Ahoy::Message" do
+    user
+  end
```

Comment:
I [deleted](https://github.com/thepracticaldev/dev.to/pull/9239/files#diff-0d047dd8a53c7a13c3d6f1867238d8d2) the `spec/factories/ahoy_message.rb` factory file since:
1) it didn't appear to be used, and
2) I didn't want to create 3 separate factories.

So, I just combined all the `Ahoy`-related factories into a single file.